types/bootstrap_token.pp

```diff
@@ -1,2 +1,2 @@
 # @summary A Kubernetes bootstrap token, must be 16-characters lowercase alphanumerical
-type K8s::Bootstrap_token = Pattern[/^[a-z0-9]{16}$/]
+type K8s::Bootstrap_token = Pattern[/^[a-z0-9]{16}\z/]
```

Comment:
I just tested with a leading newline in the token and I got the same problems. It is probably a less common mistake to make, but it would be good to fix it too. I can make a follow up PR.